packages/fxa-settings/src/components/Settings/ProductPromo/index.tsx

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import monitorTextLogo from './monitor-text-logo.svg';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import classNames from 'classnames';
+import { MonitorLink } from '../../../constants';
+import { MozServices } from '../../../lib/types';
+import { useAccount } from '../../../models';
+
+export enum ProductPromoType {
+  Sidebar = 'sidebar',
+  Settings = 'settings',
+}
+
+export interface ProductPromoProps {
+  type?: ProductPromoType;
+  // product?: MozServices;
+}
+
+export const ProductPromo = ({
+  type = ProductPromoType.Sidebar,
+}: ProductPromoProps) => {
+  const { attachedClients } = useAccount();
+
+  const hasMonitor = attachedClients.some(
+    ({ name }) => name === MozServices.Monitor
+  );
+  // const hasMonitorPlus = subscriptions.some(
```

Comment:
I'm going to leave these comments in, we have another ticket for MonitorPlus and another ticket for Glean events in the epic.